bpdv3/Assets/Scripts/Floor/TileController.cs

```diff
@@ -25,15 +25,17 @@ void Start()
     // Update is called once per frame
     void Update()
     {
+        // Specified by the GDD, on the 6th beat the floor moves until it reaches the 12th beat. Seen below **.
         if (SceneController.instance.scrollBeatCount >= 6)
```

Comment:
That 6 is called a magic number, where did it come from where does it go, what is its origin story? :D Please define it as a constant and give logical name e.g. s_BeatLimit ( I personally use s_ prefix for constants then pascal case for the rest)